curriculum/challenges/english/19-foundational-c-sharp-with-microsoft/write-your-first-code-using-c-sharp/perform-basic-operations-on-numbers-in-c-sharp.md

```diff
@@ -18,7 +18,9 @@ This challenge will be partially completed on Microsoft's learn platform. Follow
 
 What is the value of the following result?
 
-`int result = 3 + 1 * 5 / 2;`
+```clike
```

Comment:
What language slug is this?

Reply:
From chatGPT:

"In Prism.js, the clike language heading is used to define a generic language category that encompasses a group of languages that share C-like syntax. This category includes programming languages that are structurally similar to the C programming language, such as C++, C#, Java, and many others."

That was the assumption I made, but I guess I wasn't sure until now (assuming GPT isn't lying). We already had it [included in our config](https://github.com/freeCodeCamp/freeCodeCamp/blob/d31e910f3a4078e01eec87eb181c544f32a0dea4/client/.babelrc.js#L50) so I just used that.